src/tests/test-data/notes-settings.json

```diff
@@ -1,31 +1,31 @@
 [
   {
-    "id": 1,
-    "note_id": 1,
+    "id": 2,
```

Comment:
why modify this data?

Reply:
Because POST by default make primary key = 1 in database and we have 500 error. So that is why we change id to another number and therefore other test data were subsequently changed as well

Reply:
is it possible to make it lookup existing ids and increment it?

Reply:
i think there are autoincrement sequences in database, so why does POST make pkey = 1 by default?

Reply:
Idk, but if we have test data starts with id = 1 it returns 500 code

Reply:
Splited to separated issue
https://github.com/codex-team/notes.api/issues/162